bluebutton-data-pipeline-rif/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/model/CompoundCode.java

```diff
@@ -0,0 +1,53 @@
+package gov.hhs.cms.bluebutton.datapipeline.rif.model;
+
+/**
+ * Enumerate the possibly values for Compound Code for Part D
+ */
+public enum CompoundCode {
+	NOT_COMPOUNDED(1, "Not Compounded"), COMPOUNDED(2, "Compounded");
+
+	private final Integer code;
+	private final String description;
```

Comment:
Is the `description` field used by any code, or should it just be a JavaDoc comment on the constants?